lua/conquer.lua

```diff
@@ -115,22 +114,26 @@ end
 
 conquer.key_enter = function(og_bufnr)
   local lines = vim.api.nvim_buf_get_lines(0, 0, 1, false)
-  local results = vim.split(vim.fn.execute(lines), "\n")
+  local results = vim.fn.execute(lines)  -- Do not recalculate
+  local results_pretty = vim.split(vim.fn.execute(lines), "\n")
```

Comment:
Wait, doesn't this run the command twice? That'd be very unexpected (`let x = x + 1` for example).

Reply:
Yes, I forgot the update that.